### PR TITLE
Add Unit Tests for biggest_service_class with JUnit 5 and Mockito

### PR DESCRIPTION
**Pull request by tomrist9**
This PR introduces unit tests for the biggest_service_class(C) in the project using JUnit 5 and Mockito. The tests ensure that all methods in the service class are covered for positive, negative, and edge cases, following the task requirements.

### CODE REVIEW

File: andersen-homework/src/test/java/TicketServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import com.example.ticket.service.TicketService;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.MockitoAnnotations;
+import java.util.Optional;
+
+public class TicketServiceTest {
```
Comment (Darya-Drutsko) [inline andersen-homework/src/test/java/TicketServiceTest.java:9]: What we're mocking in this test?

File: andersen-homework/src/main/java/com/example/ticket/service/TicketService.java [hunk 2]
```diff
@@ -1,26 +1,36 @@
...
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
 
 @Service
 public class TicketService {
-    @Value("${classpath:tickets.json}")
-    private Resource resource;
 
+    private final Map<String, String> ticketRepository = new HashMap<>();
```
Comment (Darya-Drutsko) [inline andersen-homework/src/main/java/com/example/ticket/service/TicketService.java:12]: This class supposed to interact with db, not with in-memory hashMap, but making call for db and mocking db call in tests(call to repository)

File: andersen-homework/src/main/java/com/example/ticket/service/TicketService.java [hunk 3]
```diff
@@ -1,26 +1,36 @@
...
 
-    public List<Ticket> loadTickets() throws Exception {
-        ObjectMapper objectMapper = new ObjectMapper();
-        TypeReference<List<Ticket>> tickets = new TypeReference<List<Ticket>>() {
-        };
-        InputStream inputStream = resource.getInputStream();
-        return objectMapper.readValue(inputStream, tickets);
+
+    public boolean createTicket(String ticketId, String description) {
+        if (ticketRepository.containsKey(ticketId)) {
```
Comment (Darya-Drutsko) [inline andersen-homework/src/main/java/com/example/ticket/service/TicketService.java:16]: Here we supposed to interact with Ticket object and save Ticket into db

**Review by Darya-Drutsko — COMMENTED** [review]
According to your implementation tests are correct, but the task was related to writing tests with mocking requests for db, so we should use previous task as a base for writing tests(where we save ticket in db)
